vsts/pipelines/templates/_releaseTemplate.yml

```diff
@@ -0,0 +1,120 @@
+parameters:
+  ascName: OryxMCR
+  acrName: oryxdevmcr.azurecr.io
+  acrProdName: oryxmcr.azurecr.io
+
+steps:
+
+- checkout: self
+  clean: true
+
+- script: |
+    if [ "$(ReleaseBuildImages)" != "true" ] && [ "$(ReleaseRuntimeImages)" != "true" ]
+    then
+      echo "Invalid configuration."
+      echo "Variable 'ReleaseBuildImages' or 'ReleaseRuntimeImages' needs to be 'true' to run this"
+      exit 1
+    fi
+  displayName: 'Validate release pipeline run'
+
+- task: Docker@1
+  displayName: Container registry login
+  inputs:
+    command: login
+    azureSubscriptionEndpoint: ${{ parameters.ascName }}
+    azureContainerRegistry: ${{ parameters.acrName }}
+
+- task: DownloadBuildArtifacts@0
+  displayName: 'Download Build Artifacts'
+  inputs:
+    buildType: specific
+    project: '0bdbc590-a062-4c3f-b0f6-9383f67865ee'
+    pipeline: 11021
+    specificBuildWithTriggering: true
+    artifactName: drop
+
+- task: CopyFiles@2
+  displayName: 'Copy Files to: $(Build.ArtifactStagingDirectory)'
```

Comment:
Usually we copy files to staging directory so that when we really want to publish artifacts, we can copy the contents from the staging directory to the final directory, but here we do not do that, so do we still need this copy step? Can't we access the System.ArtifactsDirectory directly?

Reply:
Also I see that there are 2 `drop` folders now...I am wondering if the second one got created because you are copying files to staging directory here and hence it is automatically copied to the final directory?

![image](https://user-images.githubusercontent.com/6559784/59436761-18667980-8da5-11e9-8c28-0a68868f533d.png)

Reply:
as discussed .. this is something vsts needs to address ... I will address this in the community